### PR TITLE
Setup Cucumber

### PR DESCRIPTION
**Pull request by mantanquzzmo**
[PT](https://www.pivotaltracker.com/story/show/170078484)

adds cucumber support

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -20,3 +20,9 @@ production:
 
 cucumber:
   <<: *test
+
+cucumber:
```
Comment (oliverochman) [inline config/database.yml]: is the code repetition necessary here?

File: lib/tasks/cucumber.rake [hunk 2]
```diff
@@ -51,21 +51,21 @@ begin
     end
   end
   desc 'Alias for cucumber:ok'
-  task :cucumber => 'cucumber:ok'
+  task cucumber: 'cucumber:ok'
 
-  task :default => :cucumber
+  task default: :cucumber
```
Comment (oliverochman) [inline lib/tasks/cucumber.rake]: have you tried to run cucumber after changing this?

File: features/support/env.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require 'cucumber/rails'
+
+ActionController::Base.allow_rescue = false
+
+begin
+  DatabaseCleaner.strategy = :transaction
+rescue NameError
+  raise "You need to add database_cleaner to your Gemfile (in the :test group) if you wish to use it."
+end
+
```
Comment (oliverochman) [inline features/support/env.rb]: please add this line 
`World FactoryBot::Syntax::Methods
`

File: config/database.yml (hunk 1 above)
Comment (mantanquzzmo) [reply config/database.yml]: probably not. think we are in version hell.

File: lib/tasks/cucumber.rake (hunk 2 above)
Comment (mantanquzzmo) [reply lib/tasks/cucumber.rake]: yes.

File: features/support/env.rb (hunk 3 above)
Comment (mantanquzzmo) [reply features/support/env.rb]: added together with fixes in gemfile and env.rb

**Pull request merged**